pyccel/ast/type_annotations.py

```diff
@@ -128,6 +128,14 @@ def is_const(self):
         """
         return self._is_const
 
+    @is_const.setter
+    def is_const(self, val):
+        self._is_const = val
```

Comment:
Maybe check that `isinstance(val, bool)`?

Reply:
Fixed 8f0dd0d6